attestation-service/src/token/simple_default_policy.rego

```diff
@@ -28,10 +28,11 @@
 package policy
 
 import future.keywords.every
+import future.keywords.if
```

Comment:
what's going on here? was `if` not in the original simple-default-policy and should have been (so you added it)?

Reply:
Our CI does a formatting check of the default policy using the OPA package. We don't pin the version of the package so sometimes it wants new things. Recently it has started complaining about not having `if` before blocks.

The versioning of OPA is a little squirrely but it hasn't been too big of an issue so far.